src/main/java/org/opentripplanner/routing/core/State.java

```diff
@@ -41,6 +43,8 @@
     // accumulated weight up to this state
     public double weight;
 
+    public double estimatedRemainingWeight;
```

Comment:
to nie jest dobry pomysł, już lepiej jest skorzystać z tego, że mamy funkcję do liczenia tej heurystyki - która lada chwila będzie ładnie wydzielona do osobnej klasy

Reply:
Zaraz, przecież na tą zmienną nigdy nic nie przypisujesz nawet...

Reply:
1. Masz rację, zmieniłem to.
2. Podczas mergowania deva usunąłem przypisywanie przypadkiem...